app/scripts/controllers/editorial-comments.js

```diff
@@ -0,0 +1,172 @@
+
+'use strict';
+
+/**
+* AngularJS controller for managing various actions on the 
+* editorial comments, e.g. adding new comments, resolving comments etc.
+*
+* @class EditorialCommentsCtrl
+*/
+angular.module('authoringEnvironmentApp').controller('EditorialCommentsCtrl', [
+	'$scope',
+	'$timeout',
+	'editorialComments',
+	'$interval',
+	'TranslationService',
+	'toaster',
+	function (
+		$scope,
+		$timeout,
+		editorialComments,
+		$interval,
+		TranslationService,
+		toaster
+	) {
+
+
+	$scope.comments = editorialComments;
+	editorialComments.init();
+	$scope.stopRefreshing = false;
+	$scope.isLoading = false;
+    $scope.create = {};
+    $scope.isSending = false;
```

Comment:
I would consider attaching everything to the controller instance itself and using the `controllerAs` syntax.